tests/integration/kubernetes/tests_common.sh

```diff
@@ -97,6 +96,10 @@ exec_host() {
 	# Wait for the newly created pod to be ready
 	kubectl wait pod -n kube-system --timeout="30s" --for=condition=ready custom-node-debugger > /dev/null
 
+	if [ $? -ne 0 ]; then
+		return $?
+	fi
+
```

Comment:
In a bats environment this is already done automatically, no?

Outside of such an environment, may be better to use `set -e`?

Reply:
Thanks for the comment (don't know why it has been outdated). I put this because there is no `set -e`. But at the same time, I did not want to place `set -e` in the file which could make an unexpected influence throughout the file.

Reply:
Yeah, I will do it together with feedback from Wainer. Thanks a lot!